tests/testthat/test-utilities-simulation.R

```diff
@@ -0,0 +1,22 @@
+## context("initializeSimulation")
+
+test_that("`initializeSimulation()` loads a simulation at the minimum", {
+  simulation <- loadSimulation(system.file("extdata", "simple.pkml", package = "ospsuite"))
+  initializeSimulation(simulation, steadyStateTime = TRUE)
+  simulationResults <- runSimulation(simulation)
+  expect_s3_class(simulationResults, "SimulationResults")
```

Comment:
What about 

`expect_true(isOfType(simulationResults, "SimulationResults"))`

It is more straightforward because simulationResults is not actually an S3 class (or not designed to be)

Reply:
If it makes it more readable for you, I can change it, but all R6 classes also come with S3 hierarchy. 

cf. https://adv-r.hadley.nz/r6.html#introspection

![image](https://user-images.githubusercontent.com/11330453/181471959-504d171d-782b-448b-a642-7e0623aa3bdd.png)